src/components/common/calendar/Calendar.js

```diff
@@ -27,25 +26,21 @@ function CalendarApp() {
   const [eventFlag, setEventFlag] = useState(true);
   const [form] = Form.useForm();
 
-  const { authState, oktaAuth } = useOktaAuth();
-  const { idToken } = authState;
+  // const { idToken } = authState;
 
-  // const token = oktaAuth.getIdToken();
-
-  useEffect(() => {
-    if (eventFlag) {
-      console.log(authState);
-      console.log(oktaAuth);
-      axiosWithAuth(idToken)
-        .get('/calendar-events/user')
-        .then(res => {
-          setEventsArr(res.data.events);
-        })
-        .catch(err => console.error(err));
-    }
-    setEventFlag(false);
-    // eslint-disable-next-line
-  }, [eventFlag, idToken]);
+  //TO-DO: Implement Auth0
+  // useEffect(() => {
+  //   if (eventFlag) {
```

Comment:
Will this code function in its current state? It looks like the changes just remove console logs, but the rest is using state.

Reply:
I believe it would compile with an error saying that idToken was not defined, so for the most part I was commenting the whole function out, and in a few cases I set idToken to a string.  So, it was quicker for now to comment this out. The app did work when I made this PR.